django_celery_results/models.py

```diff
@@ -51,7 +51,6 @@ class TaskResult(models.Model):
                     'used with the task'))
     status = models.CharField(
         max_length=50, default=states.PENDING,
-        choices=TASK_STATE_CHOICES,
```

Comment:
the problem with this change I can see is, people using this choice field in existing projects will face breaking change. so we have to find a solution which is backward compatible and existing apps won't break.

Reply:
@auvipy wouldn't it still let them use the choices? From my understanding removing the `choices` parameter just unrestricts the choices from `TASK_STATE_CHOICES`. What kind of breaking change are you talking about here?